package.json

```diff
@@ -66,9 +66,14 @@
     "build": "react-scripts build",
     "test": "jest",
     "eject": "react-scripts eject",
-    "fix": "eslint --fix --ext js,jsx . && prettier -w .",
-    "lint": "eslint --ext js,jsx --max-warnings 0 . && prettier -c .",
-    "prepare-husky": "husky"
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint:fix": "npm run lint --fix && npm run format",
+    "prepare-husky": "husky",
+    "cy:dev": "npm start & wait-on http://localhost:3000 && STAGE_ENV=development cypress run",
+    "cy:prod": "npm run build && STAGE_ENV=production cypress run",
+    "cy:dev:open": "npm start & wait-on http://localhost:3000 && STAGE_ENV=development cypress open",
```

Comment:
We currently use two relevant CI/CD scripts for testing:
`.github/workflows/pr.yaml` runs whenever a PR is opened, while `.github/workflows/push.yaml` runs whenever a PR is approved and pushed. Could you modify these to run the relevant scripts? I'm guessing that would mean running `npm run cy:dev` inside the PR workflow and `cy:prod` in the push, but I'm unfamiliar with Cypress, so would the `open` scripts be more correct here?